README.md

```diff
@@ -1,2 +1,122 @@
-# roadmap
-Master roadmap for Project Jupyter
+# Project Jupyter Roadmap
+
+The following is a very rough outline of the roadmap for Project Jupyter and IPython.
+Historically, we have published major releases every 6-9 months. With our project now
+split across many subprojects, we expect to release subprojects separately and more
+often.
+
+This roadmap only includes mainline subproject under the Jupyter
+(https://github.com/jupyter) and IPython (https://github.com/ipython) GitHub orgs. We
+also have a number of newer, experimental subprojects in the Jupyter Incubator
+(https://github.com/jupyter-incubator) organization.
+
+## Notebook
+
+### Next (4.1)
+
+* Multi-cell selection and actions.
```

Comment:
In a very high level document like this, I don't think it is helpful to make that distinction.